### PR TITLE
[MODULAR] Gives roboticists a medkit containing synthetic treatment items, adds general synth medkits, rebalances some synth medicine-centric cargo packs, and more

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Honey, its the end of the month, time for your monthly synthetic wound PR!

Title. See proof of testing for screenshots.

## How This Contributes To The Skyrat Roleplay Experience

When I added things like chilled hercuri or analyzers to the robo vendor, they were added because I wanted robos to have them at roundstart. I have since realized I can use the outfit to give them these things. This PR does that.

There are currently _no_ normal synth medkits. This is an issue, since a synth trauma kit, being a dufflebag, is not portable. This fills the gap.

The synth burns kit was given 2 more bottles of each, to make it worth it compared to the synthetic treatment kits.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
add: Roboticists now spawn with synthetic medkits to make their jobs a bit easier
add: New cargo packs for synthetic medkits
balance: Roboticists now spawn with diag huds, welding helmets, and black gloves
balance: Synthetic burn kits now have extra bottles
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/jobs/job_types/roboticist.dm [hunk 1]
```diff
@@ -4,10 +4,21 @@
...
+
 /datum/job/roboticist
 	description = "Build cyborgs, mechs, AIs, and maintain them all. Create MODsuits for those that wish. Try to remind medical that you're \
 	actually a lot better at treating synthetic crew members than them."
 
+/datum/outfit/job/roboticist/New()
+	. = ..()
+
+	LAZYINITLIST(backpack_contents)
+	backpack_contents[/obj/item/clothing/head/utility/welding] = 1
```
Comment (nikothedude) [inline modular_skyrat/master_files/code/modules/jobs/job_types/roboticist.dm:20]: welding helmets are really important for treatment

Comment (nikothedude) [reply modular_skyrat/master_files/code/modules/jobs/job_types/roboticist.dm:20]: you cant use a welding tool without it, and im trying to maintain some level of parity between MDs and roboticists

if a MD can come off the shuttle and immediately start suturing someone, a robo should be able too as well

**Comment by YakumoChen** [comment]
Why diag huds?
I don't really care either way, but it seems odd when almost no other job spawns with a HUD item (including medical) and they're available in the vendor.

**Comment by nikothedude** [comment]
> Why diag huds? I don't really care either way, but it seems odd when almost no other job spawns with a HUD item (including medical) and they're available in the vendor.

Diag huds are crucial for treatment of T3 blunt wounds, which have a fail chance if not done by a robo without a diag hud

Vendors have them sure, but in the event of a vendor unstock station trait (i dunno the real name but you know what i mean) they can become unavailable, same with other robo items

**Comment by trefoils** [comment]
Why not add the ability to hold  drapes to these? I am also not sure I read this right but it seems they can't hold multitools? Multitools would be nice just for posi reboot surgery. I am comparing these in my head to the ones doctors spawn with the uh surgical kits. Dunno if that was your goal persay I might be over reaching.

**Comment by nikothedude** [comment]
> Why not add the ability to hold drapes to these? I am also not sure I read this right but it seems they can't hold multitools? Multitools would be nice just for posi reboot surgery. I am comparing these in my head to the ones doctors spawn with the uh surgical kits. Dunno if that was your goal persay I might be over reaching.

they can already hold drapes, but a multitool is a good point

**Review by jjpark-kb — APPROVED** [review]

**Pull request merged**